client/components/select-dropdown/index.jsx

```diff
@@ -154,6 +154,7 @@ class SelectDropdown extends Component {
 				const newChild = React.cloneElement( child, {
 					ref: ( child.type === DropdownItem ) ? 'item-' + refIndex : null,
 					key: 'item-' + index,
+					isDropdownOpen: this.state.isOpen,
```

Comment:
It is true that `isDropdownOpen` is used in the `select-dropdown` component, but the property belongs to `SelectDropdownItem` component.
In this case I wanted to make crystal clear, the fact that, the property reflects the status of the parent component (at the cost of being redundant).